tasks/elasticsearch-parameters.yml

```diff
@@ -50,7 +50,7 @@
   when: use_system_d
 #For directories we also use the {{inventory_hostname}}-{{ es_instance_name }} - this helps if we have a shared SAN.
 
-- set_fact: instance_suffix={{inventory_hostname}}-{{ es_instance_name }}
+- set_fact: instance_suffix={{ inventory_hostname + '-' + es_instance_name if (es_shared_storage | default(false)) else es_config['cluster.name'] }}
```

Comment:
This is [failing the build](https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/OS=ubuntu-1604,TEST_TYPE=standard,VERSION=5.x,label=swarm/29/console) because `es_shared_storage` isn't defined by default. I would recommend changing this to just `if es_shared_storage` and setting `es_shared_storage: False` in `defaults/main.yml`.

```
12:01:51        TASK [elasticsearch : set_fact] ************************************************
12:01:51        fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'cluster.name'\n\nThe error appears to have been in '/tmp/kitchen/elasticsearch/tasks/elasticsearch-parameters.yml': line 53, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact: instance_suffix={{ inventory_hostname + '-' + es_instance_name if (es_shared_storage | default(false)) else es_config['cluster.name'] }}\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes.  Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n\nexception type: <class 'ansible.errors.AnsibleUndefinedVariable'>\nexception: 'dict object' has no attribute 'cluster.name'"}
```